style.css

```diff
@@ -1,545 +1,853 @@
+  padding: 1rem 0;
+  color: #fff;
+}
+
+.svg2-language {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-top: 3rem;
+}
+
+#framework {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 27px;
+  color: #fff;
+}
+
+.js-section2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: 'Inter', sans-serif;
+  font-size: 10px;
+  line-height: 43px;
+  color: #fff;
+  padding-top: 2rem;
+}
+
+.svg3-language {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-top: 3rem;
+}
+
+#skills-section {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 27px;
+  color: #fff;
+}
+
+.section3 {
+  margin-bottom: 4rem;
+}
+
+.js-section3 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: 'Inter', sans-serif;
+  font-size: 10px;
+  line-height: 43px;
+  color: #fff;
+  padding-top: 2rem;
+}
+
+.form-section {
+  height: 500px;
+}
+
+.form-h2 {
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+  font-weight: bold;
+  font-size: 27px;
+  line-height: 35px;
+  margin-top: 3rem;
+  margin-bottom: 2rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+form {
+  padding: 1em;
+  margin: 0 1rem;
+  border: none;
+}
+
+.ul-for-form {
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+label {
+  width: 90px;
+  text-align: right;
+}
+
+input {
+  border: none;
+  border-bottom: 1px solid #dbd8d7;
+  padding: 1rem;
+}
+
+input,
+textarea {
+  width: 300px;
+  box-sizing: border-box;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  line-height: 24px;
+  padding-top: 1rem;
+  padding-left: 1rem;
+  margin-top: 2rem;
+}
+
+.text-input {
+  background-color: #fbf8f7;
+}
+
+::placeholder {
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  line-height: 24px;
+  color: #3c3a39;
+}
+
+.button-container {
+  margin: 1rem auto;
+}
+
+.get-button {
+  width: fit-content;
+  color: #fff;
+  background-color: #ff6b00;
+}
+
+.get-button:hover {
+  color: #ff6b00;
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  box-sizing: border-box;
+}
+
+.get-button:enabled {
+  background: #f55800;
+  color: #fff;
+}
+
+.get-button:disabled {
+  background-color: #f4f1f0;
+  color: #5b5957;
+}
+
+.form-line1 {
+  color: #dfe1e6;
+  width: 100%;
+}
+
+.social-icons2 > #form-icons2 {
+  margin-top: 3rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.social-icons2 > ul > li > a {
+  color: #505f79;
+}
+
+.form-line2 {
+  width: 12rem;
+  height: 6px;
+  background-color: #1c1a19;
+  color: #1c1a19;
+  margin: 1rem auto;
+  border-radius: 10px;
+  margin-bottom: 2rem;
+}
+
+.nav-desktop {
+  display: none;
+}
+
+.desktop-header {
+  display: none;
+}
+
+.form-h3 {
+  display: none;
+}
+
+@media screen and (min-width: 992px) {

```

Comment:
- [ ] For the desktop screens, your webpage is not responsive. The layout is not matched for all sections based on padding/margin. For this reason, there is a horizontal scroll.
![Screenshot from 2021-08-20 15-01-38](https://user-images.githubusercontent.com/65501535/130230265-5025fc04-f529-485a-abbc-25e9b5187a1b.png)